content/en/docs/operations/troubleshooting/linstor-database.md

```
@@ -44,141 +54,349 @@ If you see that something could not delete properly, it's better to investigate
-kubectl -n storage exec -ti deploy/linstor-controller -- bash
+kubectl exec -ti -n cozy-linstor deploy/linstor-controller -- bash
 ```
 
 ```bash
-root@linstor-controller-75b886bc59-cxwqd:/# piraeus-entry.sh startController
-2025-01-31 21:18:57.587 [Thread-2] INFO  LINSTOR/Controller/6bc981 SYSTEM - Shutdown complete
+root@linstor-controller-85fd6d6496-cdhbc:/# piraeus-entry.sh startController
+```
+
+You'll see the error report with the report number:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)